docs/csharp/programming-guide/main-and-command-args/command-line-arguments.md

```diff
@@ -2,75 +2,77 @@
-  
-## Example  
- The following example shows how to use command-line arguments in a console application. The application takes one argument at run time, converts the argument to an integer, and calculates the factorial of the number. If no arguments are supplied, the application issues a message that explains the correct usage of the program.  
-  
- To compile and run the application from a command prompt, follow these steps:  
-  
-1. Paste the following code into any text editor, and then save the file as  a text file with the name `Factorial.cs`.  
-  
-     [!code-csharp[csProgGuideMain#16](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class1.cs#16)]  
-  
-2. From the **Start** screen or **Start** menu, open a Visual Studio **Developer Command Prompt** window, and then navigate to the folder that contains the file that you just created.  
-  
-3. Enter the following command to compile the application.  
+> To enable command-line arguments in the `Main` method in a Windows Forms application, you must manually modify the signature of `Main` in *program.cs*. The code generated by the Windows Forms designer creates a `Main` without an input parameter. You can also use <xref:System.Environment.CommandLine%2A?displayProperty=nameWithType> or <xref:System.Environment.GetCommandLineArgs%2A?displayProperty=nameWithType> to access the command-line arguments from any point in a console or Windows application.
+
+The parameter of the `Main` method is a <xref:System.String> array that represents the command-line arguments. Usually you determine whether arguments exist by testing the `Length` property, for example:
+
+[!code-csharp[csProgGuideMain#4](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#4)]
+
+You can also convert the string arguments to numeric types by using the <xref:System.Convert> class or the `Parse` method. For example, the following statement converts the `string` to a `long` number by using the <xref:System.Int64.Parse%2A> method:
+
+```csharp
+long num = Int64.Parse(args[0]);
+```
+
+It is also possible to use the C# type `long`, which aliases `Int64`:
+
+```csharp
+long num = long.Parse(args[0]);
+```
+
+You can also use the `Convert` class method `ToInt64` to do the same thing:
+
+```csharp
+long num = Convert.ToInt64(s);
+```
+
+For more information, see <xref:System.Int64.Parse%2A> and <xref:System.Convert>.
+
+## Example
+
+The following example shows how to use command-line arguments in a console application. The application takes one argument at run time, converts the argument to an integer, and calculates the factorial of the number. If no arguments are supplied, the application issues a message that explains the correct usage of the program.
+
+To compile and run the application from a command prompt, follow these steps:
+
+1. Paste the following code into any text editor, and then save the file as  a text file with the name *Factorial.cs*.
```

Comment:
This change should be reverted.

The docs style guide calls for filenames to be *italic*. (The site engineering team is working on features so they won't be localized.)

Reply:
The change follows the style guide, it made Factorial.cs italic. I didn't get why it should be reverted ?

Reply:
My mistake. I'll :shipit: now.